Sources/SpeziLLMLocal/LLMLocalSession+Generate.swift

```diff
@@ -24,22 +24,25 @@ extension LLMLocalSession {
             return
         }
         
-        let modelConfiguration = self.schema.configuration
+        let messages = if await !self.customContext.isEmpty {
+            await self.customContext
```

Comment:
As the `customContext` is never set by SpeziLLM: I guess devs could write their own context in there and format it in whatever way they want? That also means that the dev is responsible for context handling completely on their own (e.g. `schema.injectIntoContext` would do nothing anymore?)?
I'm not 100% sure if that's really a road we want to go down on.. If yes, please document it well, not just in-line but also in updated DocC articles etc.

Reply:
Yes as described in https://github.com/StanfordSpezi/SpeziLLM/pull/75#discussion_r1876712522 we want to overwrite the current context to allow more flexibility. But yes you're right the output of the LLM will not be added to the `customContext`. 
I'll leave this one open, maybe @PSchmiedmayer can give us a direction?

Reply:
I see the motivation for providing a completely different context (`customContext`) to allow way more flexibility. However, I would argue that this goes a bit against the paradigm of SpeziLLM as a whole, we might just try to just keep one context (the existing `context`).

However, I'm sure that you and Paul thought a bit more about this implementation detail (especially considering the inflexibility of SpeziChat), so feel free to disregard my feedback.